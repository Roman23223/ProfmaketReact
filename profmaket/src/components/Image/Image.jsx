import React from 'react';

function Image({ src, alt, onClick }) {
return (
<img src={src} alt={alt} onClick={onClick} style={{ maxWidth: '90%' }} />
);
}

export default Image;