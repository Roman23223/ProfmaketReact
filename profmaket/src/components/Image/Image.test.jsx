import renderer from 'react-test-renderer';
import Image from './Image';

describe('Image', () => {
  it('should render Image component correctly', () => {
    const tree = renderer.create(<Image src="test-image.png" alt="Test Image" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});