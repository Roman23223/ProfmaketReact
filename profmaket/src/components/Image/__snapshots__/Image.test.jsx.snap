// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image should render Image component correctly 1`] = `
<img
  alt="Test Image"
  src="test-image.png"
  style={
    Object {
      "maxWidth": "90%",
    }
  }
/>
`;
